cli/src/org/partiql/shell/ShellHighlighter.kt

```diff
@@ -47,8 +49,22 @@ internal class ShellHighlighter() : Highlighter {
      * Returns a highlighted string by passing the [input] string through the [lexer] and [parser] to identify and
      * highlight tokens
      */
-    override fun highlight(reader: LineReader, input: String): AttributedString {
-        if (input.isEmpty()) return AttributedString(input)
+    override fun highlight(reader: LineReader, line: String): AttributedString {
+
+        val hasAddToGlobalEnv = line.toLowerCase().startsWith(ADD_TO_GLOBAL_ENV)
+        val input = if (hasAddToGlobalEnv) {
+            line.substring(ADD_TO_GLOBAL_ENV.length, line.length)
+        } else {
+            line
+        }
+
+        if (input.isBlank()) {
+            return if (hasAddToGlobalEnv) {
+                ADD_TO_GLOBAL_ENV_COMMAND
```

Comment:
After pulling this locally, if you type in `!add_to_global_env` and press Return immediately, there's an out of bounds exception. Seems like it's from this line. We want to color the entire line and return. If we go back to when we first added highlighting (or before highlighting), we were allowed to do this. See the suggestion that looks like it fixes this:
```suggestion
                AttributedString(line, AttributedStyle.DEFAULT.foreground(AttributedStyle.CYAN))
```